include/detectcone.hpp

```diff
@@ -17,127 +17,125 @@
-  const double V_INCREMENT_32_A = 1.33; //The first vertical angle increment for HDL-32E
-  const double V_INCREMENT_32_B = 1.34; //The second vertical angle increment for HDL-32E
-
-  // Constants for degree transformation
   const double DEG2RAD = 0.017453292522222; // PI/180.0
   const double RAD2DEG = 57.295779513082325; // 1.0 / DEG2RAD;
-
-  // Variables to handle HDL32 3 parts of message
-  uint8_t m_12_startingSensorID_32; //From which layer for the first part(12 layers) of CPC for HDL-32E
-  uint8_t m_11_startingSensorID_32; //From which layer for the second part(11 layers) of CPC for HDL-32E
-  uint8_t m_9_startingSensorID_32; //From which layer for the third part(9 layers) of CPC for HDL-32E
-  array<double, 12>  m_12_verticalAngles; //Store the 12 vertical angles for the first part (including 12 layers) of CPC for HDL-32E
-  array<double, 11> m_11_verticalAngles; //Store the 11 vertical angles for the second part (including 11 layers) of CPC for HDL-32E
-  array<double, 9> m_9_verticalAngles; //Store the 9 vertical angles for the third part (including 9 layers) of CPC for HDL-32E
-  string m_12_cpcDistance_32; //The distance string for the first part of CPC for HDL-32E    
-  string m_11_cpcDistance_32; //The distance string for the second part of CPC for HDL-32E    
-  string m_9_cpcDistance_32; //The distance string for the third part of CPC for HDL-32E
-  uint64_t m_previousCPC32TimeStamp;//The sample time of the previous CPC container belonging to a HDL-32E scan
-  uint8_t m_cpcMask_32; //The lowest 3 bits represent which part(s) of HDL-32E CPC of the same scan has been received. 0100 means the first part has arrived; 0010 means the second part has arrived; 0001 means the third part has arrived.
-  
-  // Class variables to store Lidar message
-  CompactPointCloud m_cpc;
-  odcore::base::Mutex m_cpcMutex;
-  bool m_SPCReceived;//Set to true when the first shared point cloud is received
-  bool m_CPCReceived;//Set to true when the first compact point cloud is received
-  uint32_t m_recordingYear;//The year when a recording with CPC was taken
-
-  // Class variables to save point cloud 
-  MatrixXd m_pointCloud;
-  int m_pointIndex;
-  // Define constants and thresolds forclustering algorithm
-  double m_xBoundary;
-  double m_yBoundary;
-  double m_groundLayerZ;
-  double m_coneHeight;
-  double m_connectDistanceThreshold;
-  double m_layerRangeThreshold;
-  uint16_t m_minNumOfPointsForCone;
-  uint16_t m_maxNumOfPointsForCone;
-  double m_farConeRadiusThreshold;
-  double m_nearConeRadiusThreshold;
-  double m_zRangeThreshold;
-  odcore::data::TimeStamp m_CPCReceivedLastTime;
-  double m_algorithmTime;
-  Eigen::MatrixXd m_generatedTestPointCloud;
-  // RANSAC thresholds
-  double m_inlierRangeThreshold;
-  double m_inlierFoundTreshold;
-  double m_ransacIterations;
-  double m_dotThreshold;
-  uint32_t m_senderStamp = 0;
-  Eigen::MatrixXd m_lastBestPlane;
-  
-
-
+  const double PI = 3.14159265;
```

Comment:
@arbulgazar We should use the constant provided in #include <cmath> (M_PI)

Reply:
What about `DEG2RAD` and `RAD2DEG`?

Reply:
Should be adjusted in the same way (including `constexpr` to let the compile provide these constants even during compile time for further optimizations)